app/helpers/candidates/school_helper.rb

```diff
@@ -122,14 +122,16 @@ def dlist_item(key, attrs = {}, &block)
     end
   end
 
-  def content_or_msg(content, msg = nil)
+  def content_or_msg(content, msg = nil, display_as_list: false)
```

Comment:
leave this as is, it shouldn't know about processing content - its 'thin end of the wedge' and it ends up becoming some generic renderer